.github/workflows/learn-github-actions.yml

```diff
@@ -0,0 +1,13 @@
+name: learn-github-actions
+run-name: ${{ github.actor }} is learning GitHub Actions
+on: [push]
+jobs:
+  check-bats-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
- `uses` keyword specifies that
  - this step will run `v4` of the `actions/checkout `

- `actions/checkout` 
  - checks out your repository onto the runner
  - allowing you to run scripts or other actions against your code

> 기본적으로 이건 사용해야 할 것 같다. `build `하고 `test`를 하려면 code가 우선 있어야 한다.

> check out : 소스 코드나 프로젝트의 특정 버전을 CI/CD 파이프라인 실행 환경으로 가져와서 실행 가능한 상태로 만드는 과정